extension/options.html

```diff
@@ -1,23 +1,36 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Options</title>
-  <style>
-    body: { padding: 10px; }
-  </style>
+	<title>Options</title>
+	<style>
+		body { padding: 10px;	}
+	</style>
 </head>
 
 <body>
 
-  <label>
-    Source base URL:
-    <input type="text" id="baseurl">
-  </label>
+Source base URL:
+<form action="">
+	<label>
+		<input type="radio" name="baseurl" id="googlesource" value="https://android.googlesource.com"/>
+	</label> https://android.googlesource.com
```

Comment:
the text should be in the `<label>`, otherwise clicking the text doesn't select the radio